src/librbd/AioObjectRequest.cc

```diff
@@ -278,7 +288,7 @@ void AioObjectRead<I>::send() {
   librados::AioCompletion *rados_completion =
     util::create_rados_ack_callback(this);
   int r = image_ctx->data_ctx.aio_operate(this->m_oid, rados_completion, &op,
-                                          flags, nullptr);
+                                          flags, nullptr, (this->m_trace) ? this->m_trace.get_info() : nullptr);
```

Comment:
Minor: no need for the parens around the pointer (```i.e. (this->m_trace)```) -- it would only make sense if you wrapped the entire ternary op with the parens.  comment applies throughout.